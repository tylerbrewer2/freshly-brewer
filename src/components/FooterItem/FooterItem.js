import styled from 'styled-components';

const FooterItem = styled.section`
  border-left: .5px solid #4A4A4A;
  padding-left: 30px;
  height: 200px;
`

export default FooterItem;