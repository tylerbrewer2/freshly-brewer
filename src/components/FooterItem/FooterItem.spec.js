import React from 'react';
import renderer from 'react-test-renderer';

import FooterItem from './FooterItem';

it('renders correctly', () => {
  const tree = renderer.create(
    <FooterItem />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
