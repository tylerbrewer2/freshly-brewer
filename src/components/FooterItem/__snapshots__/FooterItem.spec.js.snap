exports[`test renders correctly 1`] = `
<section
  className="sc-bdVaJa iNUpdq" />
`;
